articles/app-service-mobile/app-service-mobile-windows-store-dotnet-get-started-users.md

```diff
@@ -27,6 +27,20 @@ This tutorial is based on the Mobile Apps quickstart. You must first complete th
 ## <a name="register"></a>Register your app for authentication and configure the App Service
 [!INCLUDE [app-service-mobile-register-authentication](../../includes/app-service-mobile-register-authentication.md)]
 
+## <a name="redirecturl"></a>Add your app to the Allowed External Redirect URLs
+
+Secure authentication requires that you define a new URL scheme for your app. This allows the authentication system to redirect back to your app once the authentication process is complete. In this tutorial, we use the URL scheme _appname_ throughout. However, you can use any URL scheme you choose. It should be unique to your mobile application. To enable the redirection on the server side:
+
+1. In the [Azure portal], select your App Service.
+
+2. Click the **Authentication / Authorization** menu option.
+
+3. In the **Allowed External Redirect URLs**, enter `url_scheme_of_your_app://easyauth.callback`.  The **url_scheme_of_your_app** in this string is the URL Scheme for your mobile application.  It should follow normal URL specification for a protocol (use letters and numbers only, and start with a letter).  You should make a note of the string that you choose as you will need to adjust your mobile application code with the URL Scheme in several places.
```

Comment:
section 7.1 of the OAuth 2 spec for native apps suggests only a single `/` is needed in `url_scheme_of_your_app://easyauth.callback`
 
>“Following the requirements of [RFC3986] Section 3.2, as there is no
   naming authority for private-use URI scheme redirects, only a single
   slash ("/") appears after the scheme component.”

but assuming it's been tested and works with two slashes `://` it's probably fine as is?